.github/workflows/docker-image.yml

```
@@ -0,0 +1,30 @@
+name: Publish Docker image
+on:
+  release:
+    types: [published]
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+        with:
+          images: fearofcode/psalm
```

This should be change to
```suggestion
          images: vimeo/psalm
```
I leave it as is to be able to test it on my docker hub account, when we are ready to merge need to apply the suggestion. 
To test the image you could run `docker run --rm -it -v "${PWD}":/app -w /app fearofcode/psalm -c psalm.xml.dist`

---

Seem like creating an official image would require a lot of effort and time now.
I think it's better to publish those images under `vimeo/psalm` first and after to try to submit a request for an official image to the Docker community. 
I have never done that but looks like it's stored in a separate repository like [PHP](https://github.com/docker-library/php).

---

You could use ${{ env.GITHUB_REPOSITORY }} here if it makes testing easier